reference/under_the_hood/typotolerance.md

```diff
@@ -1,6 +1,6 @@
 # Typo tolerance
 
-MeiliSearch **is [typo tolerant](/reference/under_the_hood/typotolerance.md#typo-tolerance-rules)**; this means **it understands your search even if there are typos or spelling mistakes** to a certain extend.
+MeiliSearch **is [typo tolerant](/reference/under_the_hood/typotolerance.md#typo-tolerance-rules)**; this means **it understands your search even if there are typos or spelling mistakes** to a certain extent.
```

Comment:
It's sort of cool we have such a huge typo under the `Typo tolerance` header. What do you think about removing that last bit altogether?

```suggestion
MeiliSearch **is [typo tolerant](/reference/under_the_hood/typotolerance.md#typo-tolerance-rules)**; this means **it understands your search even if there are typos or spelling mistakes**.
```